clients/js/genapi.sh

```diff
@@ -1,15 +1,12 @@
 #!/usr/bin/env sh
 
-# curl -s http://localhost:8000/openapi.json | jq > openapi.json
 curl -s http://localhost:8000/openapi.json | python -c "import sys, json; print(json.dumps(json.load(sys.stdin), indent=2))" > openapi.json
 
 if [[ "$OSTYPE" == "darwin"* ]]; then
-  # macOS
```

Comment:
is there a reason we remove these comments?

Reply:
no, my bad. I restored them.